api/src/main/java/org/openmrs/Location.java

```diff
@@ -131,6 +133,22 @@ public void setAddress2(String address2) {
 		this.address2 = address2;
 	}
 	
+	/**
+	 * @return the startDate
+	 * @since 1.9
+	 */
+	public Date getStartDate() {
+		return startDate;
+	}
+	
+	/**
+	 * @param startDate to set to
+	 * @since 1.9
+	 */
+	public void setStartDate(Date startDate) {
```

Comment:
Hi, i completely agree with @dilanthas because i thought that if user commited a change then he/she should be able to review the change made by him/her and this is done by setData .